agent/app/service/backup.go

```
@@ -124,7 +124,7 @@ func (u *BackupService) Create(req dto.BackupOperate) error {
 	if req.Type == constant.Local {
 		return buserr.New("ErrBackupLocalCreate")
 	}
-	if req.Type != constant.Sftp && req.BackupPath != "/" {
+	if req.Type != constant.Sftp {
 		req.BackupPath = strings.TrimPrefix(req.BackupPath, "/")
 	}
 	backup, _ := backupRepo.Get(repo.WithByName(req.Name))
```

1. The line `if req.Type != constant.Sftp && req.BackupPath != "/" {` should be simplified to just `if req.BackupPath != "/" {`.

   Before change:
   ```go
   if req.Type != constant.Sftp && req.BackupPath != "/" {
       req.BackupPath = strings.TrimPrefix(req.BackupPath, "/")
   }
   ```

   After change:
   ```go
   if req.BackupPath != "/" {
       req.BackupPath = strings.TrimPrefix(req.BackupPath, "/")
   }
   ```

2. If there are no further changes needed within this function block, then these two lines are now identical and can be merged into one:

   ``` go
   backup, _ := backupRepo.Get(repo.WithByName(req.Name))
   ```

   No separate suggestion is necessary for this line unless you want it to be a direct return statement instead of returning the value with panic on failure.

If other areas of the function have been altered or additional optimizations/improvements are required beyond the provided snippet, those would need to be addressed separately.